lib/aruba/platform.rb

```diff
@@ -4,18 +4,10 @@
 # Aruba
 module Aruba
   PLATFORM_MUTEX = Mutex.new
-end
+  PLATFORM = [Platforms::WindowsPlatform, Platforms::UnixPlatform].find(&:match?)
```

Comment:
Which cop makes you change `Platform` to `PLATFORM`? I feel this file should at least define `Aruba::Platform`. Renaming may even be a breaking change.

Reply:
Constants should be `SCREAMING_SNAKE_CASE`

Reply:
I think we should disable the cop for this particular case, since we're basically setting up a class here.

Reply:
It seems a proper convoluted way of doing things, but I'll alter this change for now and rebase.